runtime/vm/jvminit.c

```diff
@@ -2150,13 +2150,14 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				j9port_control(OMRPORT_CTLDATA_VMEM_PERFORM_FULL_MEMORY_SEARCH, 0);
 			}
 
+#if JAVA_SPEC_VERSION >= 15
```

Comment:
Does this feature need to be restricted to jdk15? If not, turning it on via the option should be allowed for all versions.

Reply:
I was under the impression the feature was new for JDK15 of the RI. Having said that, it should work fine with older JCLs.

Reply:
Does the Throwable method which retrieves the extended message exist pre-JDK15?

Reply:
Only JDK15+ has the support of NPE helpful message via https://github.com/ibmruntimes/openj9-openjdk-jdk15/blob/425fa83a6d01232c99cb7eaa3f5ff9f724c7b27c/src/java.base/share/classes/java/lang/NullPointerException.java#L131

So it won't work for pre-JDK15 versions even it is on by default.

https://github.com/eclipse/openj9/issues/9844 requires JCL patch.

Reply:
> Does the Throwable method which retrieves the extended message exist pre-JDK15?

No, only JDK15+ `NullPointerException` has the method to retrieve the extended messages.

Reply:
Ok, it can be dealt with via https://github.com/eclipse/openj9/issues/9844 then.